docs/source/installation/index.rst

```diff
@@ -33,33 +33,82 @@ Download the source
 Download your preferred version from `<https://github.com/ci-group/revolve2/releases>`_.
 If you need to edit Revolve2 itself to add new features, it is recommended to instead create a fork and clone using git.
 
-------------
-Install core
-------------
-``Core`` installs all code and revolve2 packages required for optimization.
-It has only PyPI dependencies and is a pure python package::
-
-    pip install <revolve2_path>/core
-
-If you need to edit Revolve2 itself to add new features, it is recommended that you use :ref:`installation/index:Editable Mode`.
-
 --------------------------------------------
-Install supplementary packages (Optional)
+Install packages
 --------------------------------------------
-Revolve2 contains additional packages that provide extra functionality. These are fully optional.
+Revolve2 contains multiple packages that provide specific functionality. These packages contain functionality for simulations, modular robot description, hardware control and optimization / EA.
+If you need to edit components of revolve2, or you want to add new features, it is recommended that you use :ref:`installation/index:Editable Mode`.
+Manual installation with editable mode requires you to install the packages in order to avoid potential conflicts or missing dependencies.
```

Comment:
```suggestion
Manual installation with editable mode requires you to install the packages in order of dependency, so that all packages are installed in editable mode.
```